### ISSUE TITLE
Hello, thank you very much for your code

### ISSUE THREAD

**Issue by SongLongZhang** [NanqingD/DeepLabV3-Tensorflow#5]
When I train this model, the loss is always around 1.6 .I used deeplab v2 caffe's voc2012 aug data set.What data set are you using?Do you need to convert the RGB of voc label?

**Comment by NanqingD** [comment]
I think I use the same data. When creating the `.tfrecord`, `segmentation = np.array(Image.open(gt_path))` will convert the RGB to corresponding labels directly.

**Comment by SongLongZhang** [comment]
I ran the code that you just published, but the loss value keeps increasing. Do you have such a problem?May I have your contact information

**Comment by NanqingD** [comment]
With weight decay = 1, you can ignore the mean_iou before seg_loss is stable, `streaming_mean_iou` is accumulated.
```
step 8197, tot_loss = 15.798698, seg_loss = 1.011377, reg_loss = 14.787322, mean_iou: 0.035620, lr: 0.009877(0.546 sec/step)                 │··············
step 8198, tot_loss = 16.305721, seg_loss = 1.518690, reg_loss = 14.787031, mean_iou: 0.035621, lr: 0.009877(0.549 sec/step)                 │··············
step 8199, tot_loss = 15.657574, seg_loss = 0.870833, reg_loss = 14.786740, mean_iou: 0.035621, lr: 0.009877(0.556 sec/step)                 │··············
step 8200, tot_loss = 15.749551, seg_loss = 0.963102, reg_loss = 14.786449, mean_iou: 0.035633, lr: 0.009877(0.553 sec/step)                 │··············
step 8201, tot_loss = 16.652136, seg_loss = 1.865979, reg_loss = 14.786158, mean_iou: 0.035634, lr: 0.009877(0.551 sec/step)                 │··············
step 8202, tot_loss = 15.865369, seg_loss = 1.079501, reg_loss = 14.785869, mean_iou: 0.035632, lr: 0.009877(0.549 sec/step)                 │··············
step 8203, tot_loss = 16.084133, seg_loss = 1.298558, reg_loss = 14.785576, mean_iou: 0.035643, lr: 0.009877(0.550 sec/step)
```
